lab-maddy/__test__/server.test.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const server = require('../server.js');
+const superagent = require('superagent');
+
+
+describe('Testing the server file', function () {
+  afterAll((done) => {
+    server.close(done);
+  });
+
+  //POST
+  describe('POST method, /cowsay endpoint', () => {
+    test('should return a status code of 200', done => {
+      superagent.post('localhost:3000/echo')
+        .send('hello from my server!')
+        .set('Content-Type', 'text/plain')
+        .end((err, res) => {
+          expect(err).toBeNull();
```

Comment:
Looks like this test is failing. This prevents your other tests from completing I think.